browser/component_updater/brave_component_installer.cc

```diff
@@ -39,7 +37,7 @@ bool RewriteManifestFile(
   DCHECK(!public_key.empty());
 
   std::unique_ptr<base::DictionaryValue> final_manifest(manifest.DeepCopy());
-  final_manifest->SetString(extensions::manifest_keys::kPublicKey, public_key);
+  final_manifest->SetString("key", public_key);
```

Comment:
Not a big deal but we should define our own constants for these.

Reply:
the problem was using `extensions::manifest_keys::*` in android

Reply:
not sure if it really makes sense to define them again with the same values just for this

Reply:
I think there's probably a better solution here though that separates extensions from components

Reply:
https://github.com/brave/brave-browser/issues/4360